invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/ComputerTabletUploadsItem.js

```diff
@@ -0,0 +1,120 @@
+  const access_status = _get(result, "ui.access_status.title_l10n", "Open");
+  const access_status_icon = _get(result, "ui.access_status.icon", "unlock");
+  const creators = _get(result, "ui.creators.creators", []).slice(0, 3);
+  const subjects = _get(result, "ui.subjects", []);
+
+  const createdDate = result.ui?.created_date_l10n_long;
+  const description_stripped = result.ui?.description_stripped;
+  const publicationDate = result.ui?.publication_date_l10n_long;
+  const resource_type = result.ui?.resource_type?.title_l10n;
+  const title = result.metadata?.title;
+  const version = result.ui?.version;
+  const is_published = result.is_published;
+
+  const viewLink = is_published
+    ? `/records/${result.id}`
+    : `/uploads/${result.id}`;
+  const icon = is_published ? (
+    <Icon name="check" color="green" />
+  ) : (
+    <Icon name="upload" color="red" />
+  );
+
+  return (
+    <Item
+      key={index}
+      className="deposits-list-item mb-20 computer tablet only flex"
+    >
+      <div className="status-icon mr-10">
+        <Item.Content verticalAlign="top">
+          <Item.Extra>{icon}</Item.Extra>
+        </Item.Content>
+      </div>
+      <Item.Content style={{ cursor: "default" }}>
+        <Item.Extra>
+          {result.status in statuses && result.status !== "published" && (
+            <Label size="tiny" color={statuses[result.status].color}>
+              {statuses[result.status].title}
+            </Label>
+          )}
+          <Label size="tiny" color="blue">
+            {publicationDate && publicationDate} ({version && version})
+          </Label>
+          <Label size="tiny" color="grey">
+            {resource_type && resource_type}
+          </Label>
+          <Label size="tiny" className={`access-status ${access_status_id}`}>
+            {access_status_icon && (
+              <i className={`icon ${access_status_icon}`}></i>
+            )}
+            {access_status}
+          </Label>
+          <Button
+            compact
+            size="small"
+            floated="right"
+            onClick={() => editRecord()}
+          >
+            <Icon name="edit" />
+            {i18next.t("Edit")}
+          </Button>
+          {is_published && (
+            <Button compact size="small" floated="right" href={viewLink}>
+              <Icon name="eye" />
+              {i18next.t("View")}
+            </Button>
+          )}
+        </Item.Extra>
+        <Item.Header as="h2">
+          <a href={viewLink}>{title && title}</a>
+        </Item.Header>
+        <Item.Meta>
+          <div className="creatibutors">
+            <SearchItemCreators creators={creators} />
+          </div>
+        </Item.Meta>
+        {description_stripped && (
+          <Item.Description>
+            {_truncate(description_stripped, {
+              length: 350,
+            })}
+          </Item.Description>
+        )}
+        <Item.Extra>
```

Comment:
Same suggestion as below, `subjects` might be an empty array and `createdDate` might be `undefined`. Leaving the code with an empty `Item.Extra` component.

```suggestion
        { 
          (subjects || createdDate) && 
            <Item.Extra>
                ....
           </Item.Extra>
        }
```

`createdDate` is validated twice in this suggestion though.